diffrax/_custom_types.py

```diff
@@ -52,6 +52,8 @@ class Real(AbstractDtype):
 BufferDenseInfos = dict[str, PyTree[eqxi.MaybeBuffer[Shaped[Array, "times ..."]]]]
 sentinel: Any = eqxi.doc_repr(object(), "sentinel")
 
+_LA: TypeAlias = Literal["", "space-time"]
```

Comment:
This shouldn't start with an underscore if it's imported into another module. I'd suggest calling it just `LevyArea`.

Reply:
Good point, yes